packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -0,0 +1,147 @@
+import Colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Dialog from 'src/components/Dialog'
+import { calculateDecimalsToShow } from 'src/components/TokenDisplay'
+import { Namespaces } from 'src/i18n'
+import InfoIcon from 'src/icons/InfoIcon'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { tokensWithBalanceSelector, totalTokenBalanceSelector } from 'src/tokens/selectors'
+
+function TokenBalance() {
+  const tokenBalances = useSelector(tokensWithBalanceSelector)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const totalBalance = useSelector(totalTokenBalanceSelector)
+  if (tokenBalances.length === 0) {
+    return (
+      <Text style={styles.balance} testID={'TotalTokenBalance'}>
+        {localCurrencySymbol}
+        {'0.00'}
+      </Text>
+    )
+  } else if (tokenBalances.length === 1) {
+    const tokenBalance = tokenBalances[0].balance
+    return (
+      <View style={styles.oneBalance}>
+        <Image source={{ uri: tokenBalances[0].imageUrl }} style={styles.tokenImg} />
+        <View style={styles.column}>
+          <Text style={styles.balance} testID={'TotalTokenBalance'}>
+            {localCurrencySymbol}
+            {totalBalance}
+          </Text>
+          <Text style={styles.tokenBalance}>
+            {tokenBalance.toFixed(calculateDecimalsToShow(tokenBalance))} {tokenBalances[0].name}
```

Comment:
Hey, sorry I know I mentioned this but the usage has now changed, instead of exposing the decimals to show you can now just show `formatValueToDisplay(tokenBalance)`